tests/cli/test_custom.py

```diff
@@ -274,15 +274,16 @@ def mock_jupyter_notebook(args, **kwargs):
 
 
 def test_interactive_session_render_fails(monkeypatch, tmp_nbs):
+    mock_dag = Mock(side_effect=['quit'])
+    mock_start_ipython = Mock()
     monkeypatch.setattr(sys, 'argv', ['python'])
-
-    mock = Mock(side_effect=['quit'])
+    monkeypatch.setattr(interact, 'start_ipython', mock_start_ipython)
```

Comment:
I noticed the problem was with the combination of these 2 methods:
`test_interactive_session_render_fails` and `test_interactive_session_develop`.

When I ran each separately, the tests passed, but when I ran both together, the 2nd method failed (it doesn't matter which one).

Maybe there was some collision in the interact module?

Reply:
ah, I think I see why. `start_ipython` spins up an ipython session and my guess is that such action is intended to happen only once since it's a global thing. since we were calling it twice, it went nuts.

Reply:
Make sense, I think this fix is good to go!
 I saw one dependency installation here that got stuck for 5 hours for some reason.